nameko/testing/pytest.py

```diff
@@ -78,13 +78,13 @@ def pytest_addoption(parser):
 
 def pytest_load_initial_conftests():
     # make sure we monkey_patch before local conftests
-    import eventlet
-    eventlet.monkey_patch()
+    import nameko.concurrency
+    nameko.concurrency.monkey_patch()
```

Comment:
I think we could make it an optional feature.  We might even be able to make it a config option since that can be initialized before the patching.  I would still default to monkey patching everything.  Every time I have tried to work around by using import_patched, using greened modules it ends up working but very very complex, nothing I would want to explain to users.

Reply:
Perhaps an even simpler solution:  Add an option to just NOT monkeypatch.  This would then require the user to monkey patch before importing nameko themselves in their own application or import_patched or whatever they need.  This would kind of assume that if you are advanced enough to be running into monkey patch problems they can probably handle this on their own.
Having seperate options to configure WHAT is monkey patched might get to complicated/confusing anyway.

1. which do you prefer
2. should I add it here or should we just keep this in mind for future steps?

Reply:
For now I added the option to DISABLE monkey patching using the config.  The option is only available for `nameko.cli.run` since this is where problems will occur (people running complex apps using nameko).  As mentioned, when using the options users will have to call nameko.cli.run manually in their app and use `import_patched('nameko')` or do a manual monkey patch or whatever other workaround they need beforehand.

Reply:
> This would kind of assume that if you are advanced enough to be running into monkey patch problems they can probably handle this on their own.

Completely agree. Adding the ability to disable is a useful feature and a pragmatic way for Nameko to help in situations where monkey patching is a problem.